hopperi/Main.hs

```diff
@@ -0,0 +1,97 @@
+{-# LANGUAGE PatternGuards  #-}
+module Main where
+
+import Control.Arrow (left)
+import Control.Lens
+import Control.Monad.STE
+import qualified Data.Text as T
+import qualified Data.Map as Map
+import qualified Data.Vector as V
+import Text.Parsec (parse)
+
+import Data.HopperException
+import System.Console.Haskeline
+
+import Hopper.Internal.Core.Literal
+import Hopper.Internal.LoweredCore.ANF (toAnf)
+import Hopper.Internal.LoweredCore.ClosureConversion (closureConvert)
+import Hopper.Internal.LoweredCore.ClosureConvertedANF
+import Hopper.Internal.LoweredCore.EvalClosureConvertedANF as Eval
+import Hopper.Internal.Runtime.Heap
+import Hopper.Internal.Runtime.HeapRef
+import Hopper.Parsing.Term (pExpr)
+import Hopper.Parsing.Concrete (concreteToAbstract)
+
+main :: IO ()
+main =
+  let green = "\ESC[32m\STX"
```

Comment:
do any of the ansi / color terminal libs provide help with this escaping color codes stuff?